Configure multiple Forms license files (#6889)
umbraco/UmbracoDocs: 10/umbraco-forms/installation/the-licensing-model.md

The `.lic` file must be placed in the `/umbraco/Licenses` directory to be registered by Umbraco Forms. If the file isn't placed correctly, the application will automatically switch to trial mode.

### Multiple license files

> You can install multiple Umbraco Forms license files without needing to merge them. Simply place each license file in the `/umbraco/Licenses` directory (or the alternative location). Each file should start with `umbracoForms`, such as `umbracoForms.example1.lic` and `umbracoForms.example2.lic`. This allows your installation to recognize multiple licensed domains.

Reviewer: ```suggestion
You can install multiple Umbraco Forms license files without merging them. Place each license file in the `/umbraco/Licenses` directory (or an alternative location). Each file should begin with `umbracoForms`, for example, `umbracoForms.example1.lic` and `umbracoForms.example2.lic`. This setup allows your installation to recognize multiple licensed domains.
```
